### DISCUSSION TITLE
github Achievements

### DISCUSSION THREAD

**Discussion by mbjaghou** [community/community#58762] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

i have problem in github Achievements badge , i aplly all the requirement but is not show in my profile how i can fix that

**Comment by the-officialjosh** [comment, marked as answer]
Which specific badge are you referring to? It would be helpful if you could provide more details or context about the badge in question so that we can provide a more accurate and specific answer.

**Comment by Om-Vilas-Kulkarni** [comment]
**Topic** - Achievement section bug
I have 4 pull requests merged but, I didn't get pull shark badge. Please can you tell me why?

My username is @Om-Vilas-Kulkarni

![prb2](https://github.com/community/community/assets/114779868/ce5e5880-0c2b-4c46-8aa8-0c214fc8ef71)
![prb1](https://github.com/community/community/assets/114779868/a1777496-e034-4e80-af92-6c6332985c43)
